.github/actions/sparkle-appcast/action.yaml

```diff
@@ -150,15 +150,22 @@ runs:
         print -n '${{ inputs.sparklePrivateKey }}' >> eddsa_private.key
         local feed_url='${{ steps.builds.outputs.feedUrl }}'
         local arch=${${${(s:_:)feed_url:t}[2]}//x86/x86_64}
+        local sparkle_options=(
+          --verbose
+          --ed-key-file eddsa_private.key
+          --download-url-prefix '${{ inputs.urlPrefix }}/'
+          --full-release-notes-url "${feed_url//updates_*/notes_${{ inputs.channel }}.html}"
+          --maximum-versions 0
+          --maximum-deltas ${{ inputs.count }}
```

Comment:
Enclose the workflow variable in single quotes like in all other instances.

Reply:
I thought I had issues with it locally, and it wasn't enclosed in quotes in the original code, so I didn't want to touch it. I'll try to re-test on my MacBook.

Reply:
Changed.